src/desktop/src/ui/views/wallet/Dashboard.js

```diff
@@ -67,6 +79,16 @@ class Dashboard extends React.PureComponent {
         );
     };
 
+    onPaste = (e) => {
+        const data = (event.clipboardData || window.clipboardData).getData('text')
+        const parsedCDALink = parseCDALink(data);
+        if (parsedCDALink){
+            this.props.verifyCDAContent(parsedCDALink);
+            this.props.history.push('/wallet/send');
+            return e.preventDefault();
```

Comment:
Shouldn't we _always_ prevent the default event? As in shouldn't `e.preventDefault` always be called when `onPaste` callback is triggered?

Reply:
No, because we want it to fall through to the regular paste function(s) otherwise.